config.go

```diff
@@ -12,8 +12,10 @@ import (
 )
 
 type config struct {
-	Listen      string `env:"CT_LISTEN"`
-	ListenPprof string `yaml:"listen_pprof" env:"CT_LISTEN_PPROF"`
+	Listen                     string `env:"CT_LISTEN"`
+	ListenPprof                string `yaml:"listen_pprof" env:"CT_LISTEN_PPROF"`
+	ListenMetricsAddress       string `yaml:"listen_metrics_address" env: "CT_LISTEN_METRICS_ADDRESS"`
+	ListenMetricsIncludeTenant bool   `yaml:"listen_metrics_include_tenant" env: "CT_LISTEN_METRICS_INCLUDE_TENANT"`
```

Comment:
Let's maybe rename it to smth like `MetricsIncludeTenantLabel` since it really has nothing to do with listening?

Reply:
@blind-oracle cool, done